### DISCUSSION TITLE
Regarding Bluetooth support

### DISCUSSION THREAD

**Discussion by cathery** [cathery/sys-con#5] [Q&A] [answered]
~~This app will not support bluetooth functionality until either libnx provides its implementation or more info becomes available regarding the service. For more information, see:
https://switchbrew.org/wiki/Bluetooth_Driver_services~~

Edit: 
### Use MissionControl for Bluetooth support, being maintained by ndeadly
https://github.com/ndeadly/MissionControl

**Comment by kikicakes** [comment]
I know it's not ideal for others but I use my CronusMAX to use my DS4 wirelessly with sys-con, works great too.

**Comment by developersu** [comment]
Can't provide a solution for this, but it's possible to get bluetooth dongle and use it as receiver for bluetooth gamepads as it's done in this project https://github.com/nefarius/ScpToolkit

But there are a really big amount of work should be made =(

**Comment by Shiberal** [comment]
is it now feasible?
found any solution?

**Comment by y0shin** [comment]
Any news on this?

**Comment by cathery** [comment]
I'm slowly crawling my way through the Bluetooth service.

**Comment by SWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWMI** [comment]
This would be very cool. I assume we'll need a bluetooth dongle unless the switch has bluetooth hidden in the hardware somehow.

**Comment by PhiZero** [comment]
> This would be very cool. I assume we'll need a bluetooth dongle unless the switch has bluetooth hidden in the hardware somehow.

Uh, how do your joycons and pro controller interface with the Switch? Of course it has bluetooth...

**Comment by SWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWMI** [comment]
> > This would be very cool. I assume we'll need a bluetooth dongle unless the switch has bluetooth hidden in the hardware somehow.
> 
> Uh, how do your joycons and pro controller interface with the Switch? Of course it has bluetooth...

Oh yea, forgot about that.

**Comment by Dada5OOO** [comment]
Does the new "Joy Con Droid" app changes anything regarding Bluetooth implementation for sys-con?

**Comment by Vorshim92** [comment]
Why not using a bluetooth dongle on the dock?

**Comment by EzWiz21** [comment]
> Does the new "Joy Con Droid" app changes anything regarding Bluetooth implementation for sys-con?

Joy Con Droid spoofs your Android device to act and appear as an official JoyCon/Pro Controller just like some other wireless adapters. That app makes use of the Bluetooth HID profile to turn your phone into a wireless gamepad.

**Comment by kurocygnus** [comment]
> Why not using a bluetooth dongle on the dock?

Any dongle? Or need to be a specific one like 8bitdo?

**Comment by RaoufAdel** [comment]
please make it happen!  as I have a lot of Wii controller and it will be great to connect it to my switch via Bluetooth 👍

**Comment by fortem0** [comment]
Is there a way to use a Bluetooth dongle - such as one intended for use on PCs - on Switch and connect your controllers to that? I saw someone doing something similar for Xbox 360 controllers, although those apparently use WiFi to connect.

**Comment by Cilenco** [comment]
Anything where we can help you with the Bluetooth stank?

**Comment by Valeri0p** [comment, marked as answer]
Rejoice!
https://github.com/ndeadly/MissionControl
I hope some type of collaboration between the two projects will ensue 😊

**Comment by timonte53** [comment]
Ho yes a collab between sys-con and missioncontrole could be really great !

**Comment by n0k0m3** [comment]
Mission Control and sys-con can be used at the same time
Using my Xbox 360 wired with DS4 wireless at the same time, no problem at all

@cathery should add link to Mission Control for people wants to use Bluetooth (as well as reduce your roadmap workload)
